presto-native-execution/presto_cpp/main/common/tests/SystemConfigTest.cpp

```diff
@@ -48,7 +48,7 @@ class SystemConfigTest : public testing::Test {
   }
 
   void init(
-      SystemConfig& config,
+      ConfigBase& config,
```

Comment:
why change this?

Reply:
ConfigBase is the base class for SystemConfig and NodeConfig. I added a test for NodeConfig too, so had changed this to support both subclass tests.

Reply:
Can you rename this file to `ConfigTest.cpp`,  and class to `ConfigTest`?